src/backend/distributed/operations/shard_transfer.c

```diff
@@ -212,99 +182,7 @@ citus_copy_shard_placement(PG_FUNCTION_ARGS)
-					 workerNodePort)
-{
-	uint32 connectionFlag = 0;
-
-	/* we skip child tables of a partitioned table if this boolean variable is true */
-	bool optimizePartitionCalculations = true;
-	StringInfo tableSizeQuery = GenerateSizeQueryOnMultiplePlacements(shardList,
-																	  TOTAL_RELATION_SIZE,
-																	  optimizePartitionCalculations);
-
-	MultiConnection *connection = GetNodeConnection(connectionFlag, workerNodeName,
-													workerNodePort);
-	PGresult *result = NULL;
-	int queryResult = ExecuteOptionalRemoteCommand(connection, tableSizeQuery->data,
-												   &result);
-
-	if (queryResult != RESPONSE_OKAY)
-	{
-		ereport(ERROR, (errcode(ERRCODE_CONNECTION_FAILURE),
-						errmsg("cannot get the size because of a connection error")));
-	}
-
-	List *sizeList = ReadFirstColumnAsText(result);
-	if (list_length(sizeList) != 1)
-	{
-		ereport(ERROR, (errmsg(
-							"received wrong number of rows from worker, expected 1 received %d",
-							list_length(sizeList))));
-	}
-
-	StringInfo totalSizeStringInfo = (StringInfo) linitial(sizeList);
-	char *totalSizeString = totalSizeStringInfo->data;
-	uint64 totalSize = SafeStringToUint64(totalSizeString);
-
-	PQclear(result);
-	ForgetResults(connection);
-
-	return totalSize;
-}
-
-
-/*
- * CheckSpaceConstraints checks there is enough space to place the colocation
- * on the node that the connection is connected to.
- */
-static void
-CheckSpaceConstraints(MultiConnection *connection, uint64 colocationSizeInBytes)
-{
-	uint64 diskAvailableInBytes = 0;
-	uint64 diskSizeInBytes = 0;
-	bool success =
-		GetNodeDiskSpaceStatsForConnection(connection, &diskAvailableInBytes,
-										   &diskSizeInBytes);
-	if (!success)
-	{
-		ereport(ERROR, (errmsg("Could not fetch disk stats for node: %s-%d",
-							   connection->hostname, connection->port)));
-	}
-
-	uint64 diskAvailableInBytesAfterShardMove = 0;
-	if (diskAvailableInBytes < colocationSizeInBytes)
-	{
-		/*
-		 * even though the space will be less than "0", we set it to 0 for convenience.
-		 */
-		diskAvailableInBytes = 0;
-	}
-	else
-	{
-		diskAvailableInBytesAfterShardMove = diskAvailableInBytes - colocationSizeInBytes;
-	}
-	uint64 desiredNewDiskAvailableInBytes = diskSizeInBytes *
-											(DesiredPercentFreeAfterMove / 100);
-	if (diskAvailableInBytesAfterShardMove < desiredNewDiskAvailableInBytes)
-	{
-		ereport(ERROR, (errmsg("not enough empty space on node if the shard is moved, "
-							   "actual available space after move will be %ld bytes, "
-							   "desired available space after move is %ld bytes,"
-							   "estimated size increase on node after move is %ld bytes.",
-							   diskAvailableInBytesAfterShardMove,
-							   desiredNewDiskAvailableInBytes, colocationSizeInBytes),
-						errhint(
-							"consider lowering citus.desired_percent_disk_available_after_move.")));
-	}
+	ereport(ERROR, (errmsg("master_copy_shard_placement is deprecated")));
```

Comment:
Let's still forward to citus_copy_shard_placement. Hard-deprecating all master_xxx UDFS seems like it would be better as a separate PR.